test/types/index.ts

```diff
@@ -2,6 +2,7 @@
 // This file will be passed to the TypeScript CLI to verify our typings compile
 
 import * as fastify from '../../fastify'
+import { AddressInfo } from "net";
```

Comment:
`net` should be in single quotes: `'net'`

Reply:
Also, probably no semicolon (although this file sometimes has semicolons and sometimes doesn't 😕).

Reply:
I have applied a linter.